templates/base.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Регистрация в базу данных</title>
+</head>
+<body>
+    <h1>Регистрация</h1>
+    <form method="post" action="/">
+        {{ form.hidden_tag() }}
+        <p><strong>Имя:</strong> {{ form.first_name(size=20) }}</p>
+        <p><strong>Фамилия:</strong> {{ form.last_name(size=20) }}</p>
+        <p><strong>Email:</strong> {{ form.email(size=20) }}</p>
+        <p><strong>Пароль:</strong> {{ form.password(size=20) }}</p>
+        <p>{{ form.submit() }}</p>
```

Comment:
Не совсем понятно, зачем вы пишете это, если все уже есть внутри формы.
```suggestion
        {% for field in form if field.name != 'csrf_token' %}
        <p><strong>field.label</strong> {{ field(size=20) }}</p>
        <ul class="alert alert-danger">
	    {% for error in field.errors %}
	    <li>{{ error }}</li>
	    {% endfor %}
	</ul>
        {% endfor %}
        <p>{{ form.submit() }}</p>
```
Причем строки 
```html
        <ul class="alert alert-danger">
	    {% for error in field.errors %}
	    <li>{{ error }}</li>
	    {% endfor %}
	</ul>
```
Повторяют сообщения flash об ошибках. Можно оставить что-то одно. Решать вам, что и где использовать